api/README.md

```diff
@@ -0,0 +1,65 @@
+# Django setup
```

Comment:
Isn't Django set up already after following the instructions from the readme on the main page? Do we need this readme here?

Reply:
The setup file installs pipenv and node, but developers still have to run ```pipenv install``` and ```npm install``` in the respective api and client libraries. Although I can try putting this in the build script if you don't think it adds more complexity?